tests/rados/test_client_pg_access.py

```diff
@@ -34,6 +34,7 @@ def run(ceph_cluster, **kw):
     client_node = rados_obj.ceph_cluster.get_nodes(role="client")[0]
     pool_target_configs = config["verify_client_pg_access"]["configurations"]
     num_snaps = config["verify_client_pg_access"]["num_snapshots"]
+    num_objects = config["verify_client_pg_access"]["num_objects"]
```

Comment:
There are multiple suites that have this test present.

Make use of `config.get()` method to get the `num_objects`, so that in case the num objects are not present, the test case would not fail, and would proceed with fixed no of objects.

Reply:
Code modified.